ONSdigital/dp-frontend-release-calendar
# Feature/localise section headings

### What

Localise Release page section headings

English

<img width="1017" alt="Screenshot 2022-04-29 at 17 57 41" src="https://user-images.githubusercontent.com/912770/165989842-765fd0c6-3cf8-4cb3-9d55-4e76cdb549e0.png">

Welsh

<img width="1009" alt="Screenshot 2022-04-29 at 17 57 19" src="https://user-images.githubusercontent.com/912770/165989853-155f66ca-1b7d-4b6d-9ff9-6c34876f6403.png">


### How to review

How to review

 - Run dp-design-system in a separate shell with make debug
- Run this frontend controller with make debug
- Within the search results, click on a title to see the Release page
- Verify that the section headings are localised

### Who can review

Frontend / design system developers
